### PR TITLE
Add request id to logger

### PR DESCRIPTION
**Pull request by lennym**
This allows logs from a single request to be traced through log files.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -78,7 +84,10 @@ fn = function (req, res, next) {
     };
 
     if (Churchill.options.reqLogger) {
-        req.log = log;
+        req.log = function (level) {
```
Comment (easternbloc) [inline index.js]: we don't use level

**Comment by easternbloc** [comment]
lots of failing tests :(

**Comment by lennym** [comment]
Oh yeah, the extra property on all the log entries breaks things.

**Pull request merged**